parts/k8s/kubernetescustomscript.sh

```diff
@@ -187,6 +178,11 @@ fi
 if $REBOOTREQUIRED; then
   echo 'reboot required, rebooting node in 1 minute'
   /bin/bash -c "shutdown -r 1 &"
+  if [[ $OS == $UBUNTU_OS_NAME ]]; then
+      holdWALinuxAgent "unhold"
+  fi
 else
```

Comment:
this could be an elif (but that's code hygiene we don't need to block the vendor on)